.npmignore

```diff
@@ -0,0 +1,5 @@
+test
+.editorconfig
```

Comment:
Perhaps easier just to put `.*` for this and the following? Or will it override something that shouldn't be?